vertical-pod-autoscaler/e2e/common.go

```diff
@@ -143,3 +192,68 @@ func ParseQuantityOrDie(text string) resource.Quantity {
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	return quantity
 }
+
+// PodSet is a simplified representation of PodList mapping names to UIDs.
+type PodSet map[string]types.UID
+
+// MakePodSet converts PodList to podset for easier comparison of pod collections.
+func MakePodSet(pods *apiv1.PodList) PodSet {
+	result := make(PodSet)
+	if pods == nil {
+		return result
+	}
+	for _, p := range pods.Items {
+		result[p.Name] = p.UID
+	}
+	return result
+}
+
+// WaitForPodsRestarted waits until pods are restarted.
```

Comment:
Maybe "...until some pods from the list are restarted" to clarify the meaning of podList?

Reply:
Done